materialize-sql/.snapshots/TestAlterTableAddColumnTemplate

```diff
@@ -0,0 +1,4 @@
+
+	ALTER TABLE one."reserved".checkpoints ADD COLUMN
+		checkpoint TEXT NOT NULL;
```

Comment:
This example is a bit unintuitive because we don't allow `NOT NULL` columns for this migration but the way the test is written it has everything mapped to be `NOT NULL`. I've left it as-is since it's only testing the templating.